client.go

```diff
@@ -66,6 +66,16 @@ func (c *Client) Post(resource string, data url.Values) ([]byte, error) {
 	return c.do(req)
 }
 
+func (c *Client) Put(resource string, data url.Values) ([]byte, error) {
+	req, err := http.NewRequest("PUT", c.endpoint+resource, strings.NewReader(data.Encode()))
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
I decided to use an extra struct because not all values of `trello.Card` are supported in creating a new card. (Also for example types of the position diffs `float32` in `Card` vs. `string` in `AddCardOpts` - In that case the API is not very consistent…)